tests/test_sailor/test_assetcentral/test_utils.py

```diff
@@ -99,11 +99,11 @@ class TestQueryParsers:
     @pytest.mark.parametrize('test_description,value,expected_values', [
         ('single string', 'value', "'value'"),
         ('list of strings', ['value1', 'value2'], ["'value1'", "'value2'"]),
-        ('single integer', 7, '7'),
-        ('list of integers', [3, 6, 1], ['3', '6', '1']),
-        ('single float', 3.14, '3.14'),
-        ('list of floats', [3.4, 4.5], ['3.4', '4.5']),
-        ('mixed type list', ['value 1', 18., 'value2', 5], ["'value 1'", '18.0', "'value2'", '5'])
+        ('single integer', 7, "'7'"),
+        ('list of integers', [3, 6, 1], ["'3'", "'6'", "'1'"]),
+        ('single float', 3.14, "'3.14'"),
+        ('list of floats', [3.4, 4.5], ["'3.4'", "'4.5'"]),
+        ('mixed type list', ['value 1', 18., 'value2', 5], ["'value 1'", "'18.0'", "'value2'", "'5'"])
```

Comment:
since everything is quoted now unless there is a query_transformer, the test cases for numbers become less relevant but are still valid.